src/cookbook/networking/fetch-data.md

```diff
@@ -145,6 +145,60 @@ FutureBuilder<Post>(
 );
 ```
 
+## 5. Moving the fetch call out of the `build` method
+
+Although it's convenient, it's not recommended to put a call to an API in your
+`build` method.
```

Comment:
Per writing guidelines method names should be followed by parentheses: `build()`. (This should be applied on this line and a few below here.)

Reply:
Yeah, we're kinda sorta consistent about this.  :)  Fixed.